my_sprites.py

```diff
@@ -4,6 +4,11 @@
 from typing import List
 from enum import IntEnum
 
+# Two main variables that choose what direction the player is facing.
+PLAYER_RIGHT_FACING = 0
+PLAYER_LEFT_FACING = 1
```

Comment:
We don't want global variables for this. Everything should happen in the player class. How else would you do multiplayer?